deploy/helm/flightctl/templates/flightctl-networkpolicy.yaml

```diff
@@ -11,8 +11,11 @@ spec:
         - key: kubernetes.io/metadata.name
           operator: In
           values:
-          -  {{ .Release.Namespace }}
-          -  {{ default .Release.Namespace .Values.global.flightctl.internalNamespace }}
+          - {{ .Release.Namespace }}
+          - {{ default .Release.Namespace .Values.global.flightctl.internalNamespace }}
+          {{- if .Values.global.flightctl.networkPolicyAllowList }}
+          {{ toYaml .Values.global.flightctl.networkPolicyAllowList }}
```

Comment:
when deploying UI plugin, OpenShift Console needs to access the ui service - the default networkpolicy disallows access to anything but `flightctl`. The `networkPolicyAllowList` enables us to allow other namespaces as well.